gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyDirection.java

```diff
@@ -15,13 +15,13 @@ public enum DichotomyDirection {
     ES_PT("ES-PT"),
     PT_ES("PT-ES");
 
-    private final String direction;
+    private final String name;
 
     DichotomyDirection(String direction) {
```

Comment:
Could you change the name in the parameter as well ?

Reply:
Ok